Framework/src/runTPCQCPID.cxx

```diff
@@ -0,0 +1,122 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+///
+/// \file    runTPCQCPID.cxx
+/// \author  Jens Wiechula
+///
+/// \brief run TPC PID QC task, reading tracks from file for the moment
+///
+
+// IMPORTANT:
+// for some reason, if the 'customize' functions are below the other includes,
```

Comment:
Indeed. @knopers8 might be able to give more details. Is it something that should be better documented ?

Reply:
The customize() functions have to be before `#include "Framework/runDataProcessing.h"`. That header basically checks if these functions are defined by user and if so, it invokes them. There is a trick with SFINAE expressions to do that.